### DISCUSSION TITLE
Surface Pro 9 2nd Display

### DISCUSSION THREAD

**Discussion by Raffinade** [linux-surface/linux-surface#1073] [Q&A]
Hi Everyone, 
is there anybody, who can answer me 3 questions about the surface pro9. I'm looking forward to buy that device and i want to use Linux (KDE Neon). 
Things I need absolutley are Pen, Touch 2nd Monitor. 
So here are my questions, what about i cant read in the feature matrix: 
1) Does it work to plugin a 2nd Display in USB-C without problems?
2) Do the Pen and the Touchscreen work flawless?
3) What means in the feature matrix "ACPI shutdown does not work"? Does it mean, that i can´t shut down the surface?

I hope someone who has the sf pro 9 can answer with his experience or someone who know that for sure. 

LG
Raffinade

**Comment by qzed** [comment]
1. No idea.
2. Define "flawless". Touch and pen work, but may be subject to improvements.
3. Shutdown had issues, but we've recently found an in-kernel workaround, so it should work now. I've updated the feature matrix.

I don't own a SP9 so someone who owns one can probably tell you more about points 1 and 2.

**Reply by shafai** [reply]
I have ArchLinux on SP9
1. yes and it's lovely
2. they do
3. I have issues with hibernation. it doesn't wake up easily and drains the battery while in hibernation.

**Comment by petm5** [comment]
I can confirm that a display connection via USB-C works flawlessly.

**Reply by Raffinade** [reply]
Wow thx for the helpfull answers! Sounds good. So that i maybe buy that Surface. Mine is SF Pro 3 and nealry 10 years old.

Thx again
